src/main/java/com/alvarium/hash/types/HashType.java

```diff
@@ -0,0 +1,5 @@
+package com.alvarium.hash.types;
```

Comment:
It's my sense that you could just put this in `package com.alvarium.hash`. The additional `types` package seems redundant in terms of its naming -- `com.alvarium.hash.types.HashType`

Reply:
fixed in 0d3b63b1f35965acf32b53e9300a6adbc68ea290.